Build failure with Cordova 5.0.0

I tried to add this plugin to my Cordova application build with Cordova 5.0.0 and my failed with this 

error/platforms/android/src/jp/wizcorp/phonegap/plugin/wizUtilsPlugin/WizUtilsPlugin.java:33: error: incompatible types
        _webView = webView;
                   ^
  required: WebView
  found:    CordovaWebView

--- a/android/src/jp/wizcorp/phonegap/plugin/wizUtilsPlugin/WizUtilsPlugin.java
+++ b/android/src/jp/wizcorp/phonegap/plugin/wizUtilsPlugin/WizUtilsPlugin.java
@@ -30,7 +30,7 @@ public class WizUtilsPlugin extends CordovaPlugin {
     @Override
     public void initialize(org.apache.cordova.CordovaInterface cordova, org.apache.cordova.CordovaWebView webView) {
         act = cordova.getActivity();
-        _webView = webView;
+        _webView = (WebView) webView.getView();
         super.initialize(cordova, webView);
         sdk = android.os.Build.VERSION.SDK_INT;
     }

